pkgs/development/libraries/p11-kit/default.nix

```diff
@@ -16,17 +16,12 @@ stdenv.mkDerivation rec {
   outputs = [ "out" "dev"];
   outputBin = "dev";
 
-  nativeBuildInputs = [ autoreconfHook pkgconfig which ];
+  nativeBuildInputs = [ meson pkgconfig which ninja ];
   buildInputs = [ gettext libffi libiconv libtasn1 ];
```

Comment:
Do we need gettext at run time, or should this go to `nativeBuildInputs`?

Reply:
Conveniently there's info about gettext at https://p11-glue.github.io/p11-glue/p11-kit/manual/devel-building.html#devel-building-unix

Reply:
Oh, then it should actually be `libintl`.